README.md

```diff
@@ -182,6 +182,6 @@ I would like to thank MicroVerse for the support.
 

```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍
- [ ] It would be nice if you made your commit messages capitalized to make your pull request professional and descriptive :blush: